example-playbooks/ideal_token_ring/ideal_token_ring.yaml

```diff
@@ -0,0 +1,196 @@
+---
+
+- name: Gather important facts about Scylla nodes
+  hosts: scylla
+  become: true
+  gather_facts: false
+  tags:
+    - gather_facts
+  tasks:
+    - name: Get seeds list
+      shell: |
+        grep '\- seeds:' /etc/scylla/scylla.yaml | awk '{ print $NF }' | tr -d '"' | tr -d "'"
+      register: _seeds
+
+    - name: Save seeds list as a fact
+      set_fact:
+        old_seeds: "{{ _seeds.stdout }}"
+
+#    - name: Let's see our new facts
+#      debug:
+#        msg: "{{ inventory_hostname }} old seeds list is {{ old_seeds }}"
+
+    - name: Get listen address
+      shell: |
+        grep "^\s*listen_address" /etc/scylla/scylla.yaml | cut -d":" -f2 | tr -d ' '
+      register: _listen_address
+
+    - name: Save listen_address as a fact
+      set_fact:
+        listen_address: "{{ _listen_address.stdout }}"
+
+- name: Generate ideal token ring
+  hosts: scylla
+  become: true
+  gather_facts: false
+  #vars:
+# path to ideal_token_ring.py and field_cloud_access.py
+#    initial_token_script_path: /path_to_scripts/ideal_token_ring/
+  tasks:
+  #TODO make them a param passed on through variables?
+    - name: init vars to avoid calling default()
```

Comment:
There is a `local_action` module to run operations "locally" or `delegate_to: localhost`.
See more on this here: https://stackoverflow.com/questions/56048959/ansible-local-action-example-how-does-it-work

This would prevent you from all this copying nonsense.